### PR TITLE
[bitnami/nginx-ingress-controller] Allow users to enable or disable http/https service port

### PR DESCRIPTION
**Pull request by lwj5**
**Description of the change**
Allow users to enable or disable http/https with an empty service port, just by leaving the port number blank.

The kubernetes/ingress-nginx already allows this by using two flags enableHttp and enableHttps
https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/values.yaml#L370

**Benefits**

Turn on and off http/https as you like

**Possible drawbacks**

None

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/nginx-ingress-controller/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: nginx-ingress-controller
 sources:
   - https://github.com/bitnami/bitnami-docker-nginx-ingress-controller
   - https://github.com/kubernetes/ingress-nginx
-version: 7.4.9
+version: 7.5.0
```
Comment (juan131) [inline bitnami/nginx-ingress-controller/Chart.yaml:29]: You're not adding any new feature. Also, it doesn't affect the default configuration of the chart.
Therefore, I'd bump the patch version instead:
```suggestion
version: 7.4.10
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for the contribution @lwj5 ! Just a minor suggestion regarding the versioning, and we'll be ready to land your changes.

**Comment by lwj5** [comment]
Got it, thanks @juan131

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**